### DISCUSSION TITLE
How to remove this plugin?

### DISCUSSION THREAD

**Discussion by IlmirNasertdinov** [sorayuki/obs-multi-rtmp#351] [General]
I installed the plugin. Now I want to remove the plugin. I don't see file or folder obs-multi-rtmp. I reinstalled obs several times but this plugin always appears

**Comment by sorayuki** [comment]
c:\programdata\obs-studio
when run the installer, it will ask whether remove the plugin, or showing the folder it will install to
________________________________
From: IlmirNasertdinov ***@***.***>
Sent: Friday, October 27, 2023 11:20:41 PM
To: sorayuki/obs-multi-rtmp ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [sorayuki/obs-multi-rtmp] How to remove this plugin? (Discussion #351)


I installed the plugin. I don't see file or folder obs-multi-rtmp. I reinstalled obs several times but this plugin always appears

―

**Reply by IlmirNasertdinov** [reply]
Thanks. I searched in C:\Program Files\obs-studio\obs-plugins
